Add advice how to prevent jinja2 warning (#60594)
ansible/ansible: docs/docsite/rst/user_guide/playbooks_tests.rst

    {{ sample_version_var is version('1.0', operator='lt', strict=True) }}

> When using ``version`` in a playbook or role, don't use ``{{ }}`` as described in the `FAQ <https://docs.ansible.com/ansible/latest/reference_appendices/faq.html#when-should-i-use-also-how-to-interpolate-variables-or-dynamic-variable-names>`_::

Reviewer: ```suggestion
When using ``version`` in a playbook or role, don't use ``{{ }}``. For details, see the  `FAQ <https://docs.ansible.com/ansible/latest/reference_appendices/faq.html#when-should-i-use-also-how-to-interpolate-variables-or-dynamic-variable-names>`_::
```
